app/controllers/rooms_controller.rb

```diff
@@ -2,9 +2,9 @@ class RoomsController < ApplicationController
   before_action :set_room, only: [:show, :edit, :update, :destroy]
 
   def index
-    @company = current_employee.company
-    @rooms = Room.where(company_id: current_employee.company_id)
-    @rooms_array = @rooms.map { |room| [room, room.company, room.amenities, room.meetings] }
+    @company = employee_company
+    @current_employee = current_employee
+    @rooms_array = get_rooms_array
   end
 
```

Comment:
employee_company and current_employee accessible anywhere, should remove all cases of it being called in the model to assign to another variable